airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -81,7 +82,9 @@ def set_working_directory_to_root() -> None:
     os.chdir(working_dir)
 
 
-def log_git_info(ctx: click.Context) -> None:
+def log_context_info(ctx: click.Context) -> None:
+    main_logger.info(f"Running airbyte-ci version {__installed_version__}")
```

Comment:
Would love if we could add `(local install)` or `(binary install)` to this!

Reply:
it's a bit harder to get in this context, but we can currently quite well assess from the workflow logs if pipx is used to install airbyte-ci or if the bin is downloaded.